wire-runtime/src/main/java/com/squareup/wire/ProtoAdapter.java

```diff
@@ -66,6 +67,18 @@ public ProtoAdapter(FieldEncoding fieldEncoding, Class<?> javaType) {
     return new RuntimeEnumAdapter<>(type);
   }
 
+  /**
+   * Creates a new proto adapter for a map using {@code keyAdapter} and {@code valueAdapter}.
+   * <p>
```

Comment:
Hmm it doesn't have any effect for me either way. It stays in the same location and the rendered output looks the same.